Makefile

```diff
@@ -11,10 +11,15 @@ build-win:
 	find . -name '*.go' | grep -v '^\.\/vendor' | xargs gofmt -w -s
 	GOOS=windows GOARCH=amd64 CGO_ENABLED=0 go build -ldflags "-X main.Version=$(VERSION)" -o carbon-relay-ng.exe ./cmd/carbon-relay-ng
 
+build-linux:
+	cd ui/web && go-bindata -pkg web admin_http_assets
+	find . -name '*.go' | grep -v '^\.\/vendor' | xargs gofmt -w -s
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags "-X main.Version=$(VERSION)" ./cmd/carbon-relay-ng
+
 test:
 	go test ./...
 
-docker: build
+docker: build-linux
```

Comment:
this doesn't seem right. why doesn't the normal build target work? it should automatically set the correct GOOS and GOARCH (unless you're crosscompiling on a host that's not linux)

Reply:
fair enough I think. let's leave it